#! /usr/bin/env python
# Create a cfg file for getting control parameters for dmoc optimal controller

PACKAGE = 'gcop_ctrl'

from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()
gen.add("iterate"        , bool_t   , 0                 , "Iterate"     , False)
gen.add("send_trajectory"        , bool_t   , 0                 , "Send Trajectory"     , False)
gen.add("b3d_iterations"        , int_t   , 0                 , "Body3d Iterations", 300, 10, 600)
gen.add("hrotor_iterations"        , int_t   , 0                 , "Hrotor Iterations", 20, 10, 600)
gen.add("final_time"        , int_t   , 0                 , "Final Time", 4, 2, 10)


#2nd param is the name of the package this "might" be used in
exit(gen.generate(PACKAGE , "hrotor_ovs" , "HrotorOVS"))




